package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua

```diff
@@ -9,6 +9,9 @@ function M.section(form)
   if enabled then
     local s = form:section(cbi.SimpleSection, nil,
       i18n.translate('This node will automatically update its firmware when a new version is available.'))
+  else
+    local s = form:section(cbi.SimpleSection, nil,
```

Comment:
Huh, why do we even assign this to a variable... Please remove the `local s = ` (in the *if* branch too).